source/Audio.cpp

```diff
@@ -38,6 +42,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #include <stdexcept>
 #include <thread>
 #include <vector>
+#include <iostream>
```

Comment:
~~remove it and I'll pretend I didnt see it~~
tbf I also use that to test

Reply:
Yeah, I have to remember to remove them and the test data file, as well as the music

Reply:
done